frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -131,21 +131,34 @@ export const ParentSelect: React.FC<IParentSelect> = ({
         hideValidation
         required={required}
         renderMenu={(results, menuProps) => {
-          /** group the results by their desired parents */
-          const resultGroup = groupBy(
-            results.map((x: SelectOption) => {
-              return {
-                ...x,
-                parentId: x.parentId,
-              };
-            }),
-            x => x.parentId,
+          const parents = results.filter(x => !x.parentId);
+          const childless = parents.filter(
+            p => !results.find(r => Number(r.parentId) === Number(p.value)),
           );
+
+          // This assigns a specific id to childless parent agencies
+          // Headers will not be displayed for these childless parent agencies but they will be displayed as regular options and grouped
+          results = results.map(x => {
+            return {
+              ...x,
+              parentId: !!childless.find(c => Number(c.value) === Number(x.value))
+                ? -1
+                : x.parentId,
+              parent: !!childless.find(c => Number(c.value) === Number(x.value))
+                ? 'Childless'
+                : x.parent,
+            };
+          });
+
+          /** group the results by their desired parents */
+          const resultGroup = groupBy(results, x => x.parentId);
+
           const items = Object.keys(resultGroup)
             .sort()
+            .reverse()
             .map(parentId => (
               <Fragment key={parentId}>
-                {!!results.find((x: SelectOption) => x.parentId === +parentId) && (
+                {!!results.find((x: SelectOption) => x.parentId === +parentId) && +parentId !== -1 && (
```

Comment:
Do not show a header for the childless parent agencies

Reply:
Made a variable instead of hard coding -1, so it is tagged as outdated but logic is the same.